Handler/Wiki.hs

```diff
@@ -494,6 +494,7 @@ getWikiEditR project_handle language target wiki_edit_id = do
 
 getNewWikiR :: Text -> Language -> Text -> Handler Html
 getNewWikiR project_handle language target = do
+    runYDB (coPageInfo project_handle language target)
```

Comment:
It's somewhat of a joke. In math, the `coThing` is usually the dual of the `thing`. `pageInfo` fetches the page and throws an error if it doesn't exist. `coPageInfo` doesn't return anything but throws an error if the page exists.